packages/web3-eth-contract/cypress.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	screenshotOnRunFailure: false,
+	video: false,
+	e2e: {
+		// We've imported your old cypress plugins here.
+		// You may want to clean this up later by importing these.
+		setupNodeEvents(on, config) {
+			return require('./cypress/plugins/index.js')(on, config);
+		},
+		specPattern: 'test/integration/**/**/*.test.ts',
+	},
+};
```

Comment:
Is it possible to move this files to `templates` and then link to packages?

Reply:
good idea. done